adhocracy4/modules/views.py

```diff
@@ -41,6 +41,8 @@ def form_valid(self, form):
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['module'] = self.module
+        if self.module.settings_instance:
+            kwargs['settings_instance'] = self.module.settings_instance
```

Comment:
The name is slightly confusing. Maybe "module_settings_instance" is clearer. I am also not sure what the "_instance" means.